app/Components/input/input.tsx

```diff
@@ -0,0 +1,32 @@
+import style from './input.module.scss'
+
+interface Props{
+    placeholder?:string;
+    type:string;
+    width?:string;
+    height?:string;
+    padding?:string;
+    bordeRadius?:string;
+    border?:string;
+    outline?:string;
+    backgroundColor?:string;
+    color?:string
+}
+
+export default function Input(props:Props){
```

Comment:
also i think height and border radius are fixed in whole design ,
Define height and border-radius variables in styles.scss for consistency and check for other things too